spec/models/cat_spec.rb

```diff
@@ -1,5 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe Cat, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it "should validate name" do
+    cat = Cat.create(age: 1, enjoys: 'Climbing trees', image: 'https://hips.hearstapps.com/hmg-prod/images/cute-cat-photos-1593441022.jpg?crop=0.670xw:1.00xh;0.167xw,0&resize=640:*')
+    expect(cat.errors[:name]).to_not be_empty
+  end
+
+  it "should validate age" do
+    cat = Cat.create(name: 'Bob', enjoys: 'Climbing trees', image: 'https://hips.hearstapps.com/hmg-prod/images/cute-cat-photos-1593441022.jpg?crop=0.670xw:1.00xh;0.167xw,0&resize=640:*')
+    expect(cat.errors[:age]).to_not be_empty
+  end
+
+  it "should validate the cat enjoys a hobby and the minimum length is 10" do
+    cat = Cat.create(name: 'Bob', age: 1, image: 'https://hips.hearstapps.com/hmg-prod/images/cute-cat-photos-1593441022.jpg?crop=0.670xw:1.00xh;0.167xw,0&resize=640:*')
+    expect(cat.errors[:enjoys]).to_not be_empty
```

Comment:
I recommend separating the test for presence and the minimum length for enjoys into two separate tests.  This makes it so that your test is being very specific on what validation is being checked.  As it stands, only one of the validations could be working and giving you a passing test.